src/scenes/Partnerships/Edit/EditPartnership.tsx

```diff
@@ -93,7 +93,7 @@ export const EditPartnership: FC<EditPartnershipProps> = (props) => {
           refetchQueries,
         });
       }}
-      title={`Edit Partnership with ${partnership.organization.name.value}`}
+      title={`Edit Partnership with ${partnership.partner.value?.organization.value?.name.value}`}
```

Comment:
Yeah but like earlier I don't think it ever will be practically.  Don't put in the safeguard unless we need it later?

Reply:
Isn't it possible that for this (and the previous instance), `partner` or `organization` will be `canRead: false`? I don't totally understand the flow here.

Reply:
I thought you only read partnerships that you created, which means you created that partner and org and you'd have all the permissions.  But now that I say it I could be wrong.  @CarsonF any thoughts?

Reply:
Actual permissions could look very different in prod.
I'm sure it will be very common to read partnerships not created by you - in fact most will be that way.
Additionally close to 0% of our users will have actually created orgs and partners themselves.

My view almost always is if the API schema supports it we should handle it. Not everyone agrees with that mindset though.
That being said this is the "key" property of this object so it would be weird that you could see this object without seeing the _identifying_ property/relationship (similar to `LanguageEngagement.language`).

This still could result in `undefined` being displayed in the string though. This lint rule will this say this needs to be fixed. https://github.com/SeedCompany/cord-field/blob/40953541fe75a5bc747cac6fdc94d34c19c59a68/.eslintrc.yml#L27-L28

The easiest thing would be to change the title to "Edit Partnership" or leave out the "with.. " if they can't read it.
I think "with" is the wrong word/phrase too.